### ISSUE TITLE
About noisy pretraining data and model

### ISSUE THREAD

**Issue by mtt1998** [few-shot-NER-benchmark/BaselineCode#1]
Hi, thank you for sharing your code and pretrain model !

I just load the noisy pretrained linear classifier model "lc_pretrained_190.pt", and I found the output dim of final linear classifier is 9, different from 113 entity types claimed in the original paper. Is there any mistake ?  
Besides, could you share the noisy pretrained data or its download link ?

Thank you !

**Comment by teapot123** [comment]
Hi, thanks for your interest in our paper!

In the noisy supervised pre-training stage, we use meta-training paradigm to simulate the few-shot learning setting (in paper Sec 3.1), and we only randomly sample M=4 entity types at each training batch. The output class number becomes 9 because each entity type X has two labels (B-X and I-X) plus one out-of-entity label (O). This paradigm is used for both linear classifier-based and prototype-based objectives in noisy supervised pre-training.

For the noisy pre-trained data, we downloaded it from the original paper's [website](http://rali.iro.umontreal.ca/rali/?q=en/wikipedia-nrb-ner).

Thanks!

**Comment by mtt1998** [comment]
Thanks ~
